src/MergePlugin.php

```diff
@@ -366,7 +366,14 @@ public function onPostInstallOrUpdate(ScriptEvent $event)
                 $this->state->shouldOptimizeAutoloader()
             );
 
-            $installer->setUpdate(true);
+            if ($this->state->forceUpdate()) {
+              // Force update mode so that new packages are processed rather
+              // than just telling the user that composer.json and
+              // composer.lock don't match.
+              $installer->setUpdate(true);
+            } else {
+              $this->logger->log('You may need to manually run composer update to apply merge settings');
+            }
```

Comment:
Inconsistent indentation:

```suggestion
            if ($this->state->forceUpdate()) {
                // Force update mode so that new packages are processed rather
                // than just telling the user that composer.json and
                // composer.lock don't match.
                $installer->setUpdate(true);
            } else {
                $this->logger->log('You may need to manually run composer update to apply merge settings');
            }
```

Reply:
This still needs fixing at least...